grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -335,6 +335,11 @@ class StockMovementController {
                 return
             }
         }
+        if (currentLocation.isWard()) {
+            redirect(uri: "/dashboard")
+            return
+        }
+
```

Comment:
I believe you shouldn't check the location type, but the supported activities: has submit_request and doesn't have manage_inventory